lib/forklift-server/src/config.rs

```diff
@@ -192,6 +196,7 @@ fn buck2_development(config: &mut ConfigFile) -> Result<()> {
 
     config.audit.pg.certificate_path = Some(postgres_cert.clone().try_into()?);
     config.audit.pg.dbname = audit_logs::database::DBNAME.to_string();
+    config.enable_audit_logs_app = true;
```

Comment:
Worth calling out: this will enable audit logs being written to the db for local dev.

Reply:
No more!